packages/codegen-ui/lib/__tests__/generate-form-definition/generate-form-definition.test.ts

```diff
@@ -107,6 +107,130 @@ describe('generateFormDefinition', () => {
     });
   });
 
+  describe('form with FileUploaderFields', () => {
+    it('should generate form definition with default values from minimal config', () => {
+      const result = generateFormDefinition({
+        form: {
+          name: 'CreateProductForm',
+          dataType: {
+            dataSourceType: 'DataStore',
+            dataTypeName: 'Product',
+          },
+          formActionType: 'create',
+          fields: {
+            imgKeys: {
+              label: 'Images',
+              inputType: {
+                type: 'FileUploaderField',
+                fileUploaderConfig: {
+                  accessLevel: 'public',
```

Comment:
Can you remove the entire `fileUploaderConfig` for this test?? I know technically, customers would have these. But we also use the `getFormDefinitionInputElement` in the UI to generate previews. in that case, we would pass in something like `inputType: {type: 'FileUploaderField'}` and nothing else, expecting defaults to be applied.

Reply:
I want to keep this test. But I'll add a new one to cover the scenario ^.